### PR TITLE
Moved debug field to summary and defaulting targetOS to users client OS

### PR DESCRIPTION
**Pull request by jak-atx**
Fixes 
https://github.com/vmware/vic/issues/6353
https://github.com/vmware/vic/issues/6357

### CODE REVIEW

**Comment by cristianfalcone** [comment]
The code looks good to me, but shouldn't we consider the last [mockup](https://vmware.invisionapp.com/share/GDC9QEDAZ#/screens/239666195)? The debug selector position is different in that screen.

File: ui/vic-ui-h5c/vic/src/vic-webapp/src/app/create-vch-wizard/summary/summary.component.ts [hunk 1]
```diff
@@ -198,6 +209,10 @@ export class SummaryComponent implements OnInit {
           };
         }
       });
+
+    // debug
+    results['debug'] = this.form.get('debug').value;
```
Comment (cristianfalcone) [inline ui/vic-ui-h5c/vic/src/vic-webapp/src/app/create-vch-wizard/summary/summary.component.ts:226]: I think we lost the debug option in the CLI command. Not able to see it. And now I think we should update the CLI command on debug option selection.

**Comment by jak-atx** [comment]
Good catches. I must have been viewing an older mockup.

**Review by cristianfalcone — APPROVED** [review]
LGTM

**Pull request merged**